tutor/config.py

```diff
@@ -26,6 +26,15 @@ def load(root: str) -> Config:
     return load_full(root)
 
 
+def load_defaults(root: str) -> Config:
+    """
+    Load default configuration.
+    """
+    config = get_user(root)
```

Comment:
We should not be loading user config here. We only need the defaults. Thus the `root` argument is unnecessary.